spec/codecs/cef_spec.rb

```diff
@@ -452,7 +452,7 @@ def validate(e)
       end
     end
 
-    let (:trim_additional_fields_with_dot_notations) {'CEF:0|security|threatmanager|1.0|100|trojan successfully stopped|10|src=10.0.0.192 dst=12.121.122.82 ad.field[0]=field0 ad.name[1]=new_name'}
+    let (:trim_additional_fields_with_dot_notations) {'CEF:0|security|threatmanager|1.0|100|trojan successfully stopped|10|src=10.0.0.192 ad.Authentification=MICROSOFT_AUTHENTICATION_PACKAGE_V1_0 ad.Error_,Code=3221225578 dst=12.121.122.82 ad.field[0]=field0 ad.name[1]=new_name'}
```

Comment:
@nich07as I checked the link you provided to the spec for CEF  and could not find anything about a prefix `ad.`. My question is, why should this generic CEF codec have any special handling (removal of fields) of fields with this prefix, if it is not mentioned in the spec?

My understanding of the spec is, that the ArcSight products ESM and Logger do provide very limited handling for custom extensions. That is fine, but this is not a requirement for all software which is processing CEF encoded data. In fact it could be the reason, why someone uses Logstash in front of one of the mentioned ArcSight products, because access to this information is needed.

If a user really needs these fields, prefixed with `ad.` to be removed, there are filter plugins available to perform this task. In my opinion it should not be the task of the decode function of the CEF codec.